app/packages/core/src/plugins/SchemaIO/components/NativeModelEvaluationView/Evaluation.tsx

```diff
@@ -1757,3 +1759,52 @@ type PLOT_CONFIG_TYPE = {
 type PLOT_CONFIG_DIALOG_TYPE = PLOT_CONFIG_TYPE & {
   open?: boolean;
 };
+
+type CustomMetric = {
+  label: string;
+  value: any;
+};
+
+type CustomMetrics = {
+  [operatorName: string]: CustomMetric;
+};
+
+type SummaryRow = {
+  id: string;
+  property: string;
+  value: any;
+  compareValue: any;
+  lesserIsBetter: boolean;
+  filterable: boolean;
+  active: boolean;
+  hide: boolean;
+};
+
+function formatCustomMetricRows(evluationMetrics, comparisonMetrics) {
+  const results = [] as SummaryRow[];
+  const customMetrics = _.get(
+    evluationMetrics,
+    "custom_metrics",
+    {}
+  ) as CustomMetrics;
+  for (const [operatorName, customMetric] of Object.entries(customMetrics)) {
+    const compareValue = _.get(
+      comparisonMetrics,
+      `custom_metrics.${operatorName}.value`,
+      null
+    );
+    const hasComparisonValue = compareValue !== null;
+
+    results.push({
+      id: operatorName,
+      property: customMetric.label,
+      value: customMetric.value,
+      compareValue,
+      lesserIsBetter: true, // TODO - this needs to be provided from the customMetrics object
```

Comment:
What does `lesserIsBetter` do?

Reply:
```
const showTrophy = lesserIsBetter
                      ? difference < 0
                      : difference > 0;
```

Its used to show the trophy icon. It could also be used in the future to show red vs green when displaying numbers.